File: examples/number_conversion.cpp
#include<iostream>

using namespace std;

int main() {
    int four = 4;
    int three = 3;

    double half = 0.5;

    cout << "add int and double   : 4 + 0.5 = " << four + half  << endl;
    cout << "divide int and int   : 4 / 3   = " << four / three << endl;
    cout << "divide int by double : 4 / 0.5 = " << four / half  << endl;
    cout << "divide double by int : 0.5 / 4 = " << half / four  << endl;
}

File: examples/number_conversion.rs
fn main() {
    let four: u32 = 4;
    let three: u32 = 3;

    let half: f64 = 0.5;

    println!("add u32 and f64    : 4 + 0.5 = {}", four + half);
    println!("divide u32 and u32 : 4 / 3   = {}", four / three);
    println!("divide f64 and u32 : 0.5 / 4 = {}", half / four);
    println!("divide u32 by f64  : 4 / 0.5 = {}", four / half);
} 
